namerd/main/src/main/scala/io/buoyant/namerd/NamerdAdmin.scala

```diff
@@ -21,15 +23,116 @@ object NamerdAdmin {
     Handler("/config.json", new ConfigHandler(nc, NamerdConfig.LoadedInitializers.iter))
   )
 
-  def dtabs(dtabStore: DtabStore, namers: Map[Path, Namer]) = Seq(
-    Handler("/", new DtabListHandler(dtabStore)),
-    Handler("/dtab/delegator.json", new DelegateApiHandler(ns => ConfiguredNamersInterpreter(namers.toSeq))),
-    Handler("/dtab/", new DtabHandler(dtabStore))
-  )
+  def dtabs(dtabStore: DtabStore, namers: Map[Path, Namer], adminHandler: NamerdAdmin) = {
+    val adminFilter = new NamerdFilter(adminHandler)
+    Seq(
+      Handler("/", adminFilter.andThen(new DtabListHandler(dtabStore))),
+      Handler("/dtab/delegator.json", new DelegateApiHandler(ns => ConfiguredNamersInterpreter(namers.toSeq))),
+      Handler("/dtab/", adminFilter.andThen(new DtabHandler(dtabStore)))
+    )
+  }
+
+  def logging(adminHandler: NamerdAdmin): Seq[Handler] = {
+    val adminFilter = new NamerdFilter(adminHandler)
+    Seq(
+      Handler("/logging.json", new LoggingApiHandler()),
+      Handler("/logging", adminFilter.andThen(new LoggingHandler(adminHandler)))
+    )
+  }
+
+  def apply(nc: NamerdConfig, namerd: Namerd): Seq[Handler] = {
+    val handler = new NamerdAdmin(Seq(NavItem("dtabs", "dtabs"), NavItem("logging", "logging")))
```

Comment:
The `dtabs` and `Namerd` nav items link to the same place and I think are redundant